tests/assertions.ts

```diff
@@ -50,4 +51,48 @@ export async function assertGeneratedCorrectly({
 
     expect(await fse.pathExists(pathToFile), `${pathToFile} exists`).toBe(true);
   }
+
+  await matchesFixture('.prettierrc.js', { cwd: projectRoot });
+}
+
+export async function matchesFixture(
+  /**
+   * Project-relative file to test against
+   */
+  testFilePath: string,
+  options?: {
+    /**
+     * Which fixture set to use
+     */
+    scenario?: string;
```

Comment:
same comment as below re: scenario. It feels like it's better DX to just make sure that people specify it

Reply:
I disagree, tbh